src/api/configureApi.ts

```diff
@@ -26,11 +26,23 @@ import { createParticipantsRouter } from './participantsRouter';
 import { createUsersRouter } from './usersRouter';
 
 const BASE_REQUEST_PATH = '/api';
-function bypassHandlerForPaths(middleware: express.Handler, ...paths: string[]) {
+const BYPASS_PATHS = [
+  `/favicon.ico`,
+  `${BASE_REQUEST_PATH}/`,
+  `${BASE_REQUEST_PATH}/metrics`,
+  `${BASE_REQUEST_PATH}/health`,
+  `${BASE_REQUEST_PATH}/keycloak-config`,
+];
+
+function bypassHandlerForPaths(middleware: express.Handler, ...paths: (string | string[])[]) {
   return function (req, res, next) {
-    const pathCheck = paths.some((path) => path === req.path);
-    if (pathCheck) {
-      next();
+    const bypassPath = paths.find((path) =>
+      Array.isArray(path) ? path[0] === req.path : path === req.path
```

Comment:
If you map all the paths which are just strings to objects and give them a default `.method='GET'` then this logic will become simpler and you won't have to check the type multiple times.